src/main/webapp/app/exercises/quiz/manage/quiz-exercise-update.component.html

```diff
@@ -0,0 +1,338 @@
+                                }
+                                @if (pendingChangesCache && quizIsValid && !isSaving) {
+                                    <span jhiTranslate="artemisApp.quizExercise.edit.pendingChanges" class="badge bg-secondary"> </span>
+                                }
+                                @if (isSaving) {
+                                    <span jhiTranslate="artemisApp.quizExercise.edit.saving" class="badge bg-secondary"> </span>
+                                }
+                                <ng-template #tooltipTranslate>
+                                    @for (reason of computeInvalidReasons(); track reason) {
+                                        <div>
+                                            @if ((reason.translateValues | json) !== '{}') {
+                                                <p
+                                                    jhiTranslate="{{ reason.translateKey }}"
+                                                    [translateValues]="{ index: reason.translateValues.index, threshold: reason.translateValues.threshold }"
+                                                ></p>
+                                            }
+                                            @if ((reason.translateValues | json) === '{}') {
+                                                <p jhiTranslate="{{ reason.translateKey }}"></p>
+                                            }
+                                        </div>
+                                    }
+                                </ng-template>
+                                <ng-template #warningTranslate>
+                                    @for (warning of computeInvalidWarnings(); track warning) {
+                                        <div>
+                                            @if ((warning.translateValues | json) !== '{}') {
+                                                <p jhiTranslate="{{ warning.translateKey }}" [translateValues]="{ index: warning.translateValues.index }"></p>
+                                            }
+                                            @if ((warning.translateValues | json) === '{}') {
+                                                <p jhiTranslate="{{ warning.translateKey }}"></p>
+                                            }
+                                        </div>
+                                    }
+                                </ng-template>
+                                @if (!quizIsValid) {
+                                    <span class="badge bg-danger" [ngbTooltip]="tooltipTranslate" tooltip-placement="top-right auto">
+                                        <fa-icon [icon]="faExclamationCircle"></fa-icon>
+                                        <span jhiTranslate="artemisApp.quizExercise.edit.invalidInput"></span>
+                                        <span>({{ computeInvalidReasons().length }})</span>
```

Comment:
Can we avoid this method calls somehow?

Reply:
As discussed in Slack:
I renamed this file, hence it looks like I changed something here. And since this notification about invalid Input is dependent on changed form values and would require a lot of refactoring. We decided for this to be out of scope for this PR.